### PR TITLE
Fixed some styles that looked odd for release notes and add 1.1 release ...

### PR DESCRIPTION
**Pull request by skottler**
...notes to the site

### CODE REVIEW

**Comment by domcleal** [comment]
The green stable release button could be linked to these release notes now.

**Comment by skottler** [comment]
Good catch - added a new commit with that change.

-Sam

File: _includes/manuals/1.1/1.2_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
+﻿## Release notes for 1.1-1
```
Comment (domcleal) [inline _includes/manuals/1.1/1.2_release_notes.md]: Double check the original file, there's a strange unicode character in here which prevents the heading from working:

```
+<feff>## Release notes for 1.1-1
```

**Comment by domcleal** [comment]
Generally +1, it's an improvement.  I'm only a bit hesitant because there's so much of it.. what do you think about combining all the RCs together so it's just one set of changes?

File: _includes/manuals/1.1/1.2_release_notes.md (hunk 1 above)
Comment (skottler) [inline _includes/manuals/1.1/1.2_release_notes.md]: I'll fix that.

**Comment by skottler** [comment]
Yeah, unfortunately it means there are a lot of additional items in the left menu. My concern about combining all the sections into a single one is that people who are on RC3, for example, will want to know the changes between their version and 1.1.1 stable. What about putting the release notes for each RC under an additional section called "RC notes" or something so they are accessible still?

**Comment by domcleal** [comment]
Indeed, it's just trying to find a good balance - I think that's a good suggestion.  Would you mind doing that?

**Comment by skottler** [comment]
Updated and ready for a final review @domcleal!

**Comment by domcleal** [comment]
:+1: merge!  I like the CSS change too.

**Pull request merged**